Code/Framework/AzNetworking/AzNetworking/DataStructures/TimeoutQueue.h

```diff
@@ -30,7 +29,7 @@ namespace AzNetworking
     {
     public:
 
-        struct AZNETWORKING_API TimeoutItem
+        struct  TimeoutItem
```

Comment:
nitpick extra space between `struct` and `TimeoutItem`
```suggestion
        struct TimeoutItem
```